client/lib/signup/step-actions/index.js

```diff
@@ -286,6 +308,10 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 		return;
 	}
 
+	if ( newSiteParams.options.enable_fse ) {
+		recordTracksEvent( 'calypso_fse_enrolled', { flow: flowToCheck } );
```

Comment:
We analyse tracks events on a per-user basis, but this event gets recorded once per-site. Has that created any confusion in the `/new` rollout?
For instance
- user creates a site
- their ID % 100 is > 10 so they're not enrolled
- we don't record tracks event
- user creates another site at a later date
- their ID % 100 is > 50 (we've now rolled out FSE to more users) so they are enrolled
- we do record tracks event

When analysing how this user interacts with their first site, we might incorrectly think they're an FSE user because they have recorded a `calypso_fse_enrolled` event at _some_ point.

Maybe we've limited our analyse to users who only have a single site, so this hasn't been a problem.

Reply:
I see that it was added to `/new` in #60328 and it just fires as soon as you enter the flow, regardless of whether the user successfully creates a site. So I see it's just a rough guide, so probably don't need to overthink it here.

Reply:
> When analysing how this user interacts with their first site, we might incorrectly think they're an FSE user because they have recorded a calypso_fse_enrolled event at some point.

That's true, we haven't made efforts to differentiate these in a better way, but I'd imagine it's not a large fraction of users/sites. I'm open to better ideas for how to deal with this in a more precise way. My intuition is that similar effects are in play for non-enrolled users, so on average they shouldn't bias the stats one way or the other. 🤔